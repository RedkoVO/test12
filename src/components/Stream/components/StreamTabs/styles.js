export default () => ({
  root: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    color: '#fff',
    minWidth: '830px',
    margin: '25px'
  }
})
