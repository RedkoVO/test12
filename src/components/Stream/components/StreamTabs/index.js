import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import StreamTabsBottom from './components/StreamTabsBottom'

import styles from './styles'

const StreamTabs = ({ classes, streamLive }) => (
  <div className={classes.root}>
    <div className={classes.streamTabsTop}>
      <div href="#" className={classes.streamTabsUpcom}>
        UPCOMING GAMES
      </div>
      <div href="#" className={classes.streamTabsResult}>
        RESULTS
      </div>
    </div>

    <StreamTabsBottom streamLive={streamLive} />
  </div>
)

StreamTabs.propTypes = {
  classes: PropTypes.object,
  streamLive: PropTypes.array,
}

export default withStyles(styles)(StreamTabs)
