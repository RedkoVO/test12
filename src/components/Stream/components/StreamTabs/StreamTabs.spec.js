import * as React from 'react'
import renderer from 'react-test-renderer'

import StreamTabs from './'

it('renders correctly', () => {
  const tree = renderer.create(<StreamTabs />).toJSON()
  expect(tree).toMatchSnapshot()
})
