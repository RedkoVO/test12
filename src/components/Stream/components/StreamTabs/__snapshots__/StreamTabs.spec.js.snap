// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="StreamTabs-root-1"
>
  <div
    className="StreamTabs-streamTabsTop-2"
  >
    <div
      className="StreamTabs-streamTabsUpcom-3"
      href="#"
    >
      UPCOMING GAMES
    </div>
    <div
      className="StreamTabs-streamTabsResult-4"
      href="#"
    >
      RESULTS
    </div>
  </div>
  <div
    className="StreamTabsBottom-root-5"
  >
    <div
      className="StreamTabsNav-root-6"
    >
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-liveNow-8"
      >
        LIVE NOW 
        <span
          className={undefined}
        >
          2
        </span>
      </div>
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-allGames-9"
      >
        ALL GAMES 
        <span
          className={undefined}
        >
          2
        </span>
      </div>
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-dota2-10"
      >
        DOTA 2
        <span
          className={undefined}
        >
          2
        </span>
      </div>
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-starcraft2-11"
      >
        STARCRAFT 2 
        <span
          className={undefined}
        >
          2
        </span>
      </div>
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-overwatch-12"
      >
        OVERWATCH 
        <span
          className={undefined}
        >
          2
        </span>
      </div>
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-rainbowSix-13"
      >
        RAINBOW SIX 
        <span
          className={undefined}
        >
          2
        </span>
      </div>
      <div
        className="StreamTabsNav-streamTabsNavItem-7 StreamTabsNav-csGo-15 StreamTabsNav-streamTabsNavActive-14"
      >
        CS:GO
        <span
          className={undefined}
        >
          2
        </span>
      </div>
    </div>
    <div
      className="StreamLive-root-16"
    >
      <div
        className={undefined}
      >
        <div
          className=""
        >
          Live
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          12:30 PM
           
          <br />
          <span
            className="dateDay"
          >
            7/12/2018
          </span>
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          Live
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          12:30 PM
           
          <br />
          <span
            className="dateDay"
          >
            7/12/2018
          </span>
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          Live
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          12:30 PM
           
          <br />
          <span
            className="dateDay"
          >
            7/12/2018
          </span>
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          Live
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          12:30 PM
           
          <br />
          <span
            className="dateDay"
          >
            7/12/2018
          </span>
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          Live
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
      <div
        className={undefined}
      >
        <div
          className=""
        >
          12:30 PM
           
          <br />
          <span
            className="dateDay"
          >
            7/12/2018
          </span>
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            NRG
          </div>
          <div
            className=""
          >
            France
          </div>
        </div>
        <div
          className={undefined}
        >
          6.14
        </div>
        <div
          className={undefined}
        >
          VS
        </div>
        <div
          className={undefined}
        >
          1.25
        </div>
        <div
          className=""
        >
          <div
            className={undefined}
          >
            LDLC
          </div>
          <div
            className=""
          >
            Spain
          </div>
        </div>
        <div
          className={undefined}
        >
          EVENT
        </div>
        <div
          className={undefined}
        />
      </div>
    </div>
  </div>
</div>
`;
